connectors/rdbms/r2dbc/build.gradle.kts

```diff
@@ -22,17 +15,19 @@ tasks.withType<Test>().configureEach {
 }
 
 dependencies {
-    implementation(project(":core"))
-
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${Version.Coroutine}")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactive:${Version.Coroutine}")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:${Version.Coroutine}")
-
+    implementation(RiverCore)
+    Dependencies.Coroutines.forEach { implementation(it) }
+    Dependencies.R2dbc.forEach { implementation(it) }
```

Comment:
`h2` is being added here as an implementation dependency, it should be available only for testing, right?

Reply:
Right! 🚀

Reply:
@gabfssilva Is it better separate this 2 dependencies? Removing the list...

Reply:
In this specific case, I think so. But it's cool to group them and all use the same configuration. In this case, one is `implementation` & the other is `testImplementation`.